### PR TITLE
Gao/cs 315 default to safepass

### PR DESCRIPTION
**Pull request by gokeji**
## Changes:

- After permissions have been asked, route to safepass regardless of permission errors
- As long as the user has opted in before, always route to safepass when cold launching app. The SafePass will have all the surfaces for opting out, as well as fixing error states.
- Point dev builds to staging by default
- Added javascript interface to the existing `WebViewController`, and added ability to hide its close button
- Bump version number to v1.5.0

### CODE REVIEW

File: safetrace-app/Features/Common/WebViewController.swift [hunk 1]
```diff
@@ -89,6 +130,95 @@ final class WebViewController: UIViewController {
...
+                let citizenURL = Constants.citizenDeeplinkUrl
+                let isCitizenInstalled = UIApplication.shared.canOpenURL(citizenURL)
+
+                let javascriptToRun: String = "_tracing.setIsCitizenInstalled(\(isCitizenInstalled))"
+                runJavaScript(javascriptToRun)
+            } else if body == "optInToTracing" {
+                environment.safeTrace.startTracing()
+            }
+        case "user":
+//            if body == "getLocation" {
```
Comment (maxmamis) [inline safetrace-app/Features/Common/WebViewController.swift:164]: why is this here?

File: safetrace-app/Services/AppEnvironment.swift [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+
 struct AppEnvironment: Environment {
     var bluetoothPermissions: BluetoothPermissionsProviding = BluetoothPermissionsProvider()
     var notificationPermissions: NotificationPermissionsProviding = NotificationPermissionsProvider()
     var safeTrace: SafeTraceProviding = SafeTraceProvider()
     var citizen: CitizenProviding = CitizenProvider()
     var analytics: AnalyticsTracking = AnalyticsTracker()
+
+    init() {
+        if Bundle.main.bundleIdentifier == "org.ctzn.safetrace-dev" {
```
Comment (maxmamis) [inline safetrace-app/Services/AppEnvironment.swift:11]: why are you using this instead of `#if INTERNAL`?

Comment (gokeji) [reply safetrace-app/Services/AppEnvironment.swift:11]: Oh true, I can do that instead. Forgot that internal builds use the `staging` build config instead of `release`, so they have the `INTERNAL` flag

File: safetrace-app/Features/Common/WebViewController.swift (hunk 1 above)
Comment (gokeji) [reply safetrace-app/Features/Common/WebViewController.swift:164]: I can remove this. Was initially assuming that location is coming right up

Comment (maxmamis) [reply safetrace-app/Features/Common/WebViewController.swift:164]: ah, got it, no need to remove i guess

**Comment by maxmamis** [comment]
approved, let's change the bundleID check to #if INTERNAL and it's good to go

File: safetrace-app/Services/AppEnvironment.swift (hunk 2 above)
Comment (gokeji) [reply safetrace-app/Services/AppEnvironment.swift]: Addressed
